RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByIsolation.cc

```diff
@@ -500,7 +503,22 @@ double PFRecoTauDiscriminationByIsolation::discriminate(const PFTauRef& pfTau) c
     double neutralPt = 0.;
     double weightedNeutralPt = 0.;
     for (auto const& isoObject : isoCharged_) {
-      chargedPt += isoObject->pt();
+      //-------------------------------------------------------------------------
+      // CV: fix for Phase-2 HLT tau trigger studies
+      //    (pT of PFCandidates within HGCal acceptance is significantly higher than track pT !!)
+      if (enableHGCalWorkaround_) {
+        double trackPt = (isoObject->bestTrack()) ? isoObject->bestTrack()->pt() : 0.;
+        double pfCandPt = isoObject->pt();
+        if (pfCandPt > trackPt) {
+          chargedPt += trackPt;
+          neutralPt += std::max(0., pfCandPt - trackPt);
```

Comment:
```suggestion
          neutralPt += pfCandPt - trackPt;
```
the max seems superfluous given `if (pfCandPt > trackPt) `